build/status/N2FfdGVtcF93cXBfbXVuZ2Uvb3V0L2FsbF93cXBfZGF0X2xpbmtlZC5mZWF0aGVyLmluZA.yml

```diff
@@ -2,9 +2,9 @@ version: 0.3.0
 name: 7a_temp_wqp_munge/out/all_wqp_dat_linked.feather.ind
 type: file
 hash: 52efe208394fc69014a436993c482131
-time: 2019-05-09 14:28:22 UTC
+time: 2019-07-03 20:13:59 UTC
 depends:
-  7a_temp_wqp_munge/tmp/all_wqp_dat.rds.ind: 7fe951654e7d640d6487863cbc78f5b5
+  7a_temp_wqp_munge/tmp/all_wqp_dat.rds.ind: 2b24448d789fc14cd8bc78f9f60ae842
```

Comment:
I'm having a hard time understanding why the WQP data was rebuilt. This indicates that the `all_wqp_dat.rds.ind` has a new hash, but there's no new build status file for that target. Is there a modified build status file for `7a_temp_wqp_munge/tmp/all_wqp_dat.rds.ind` that needs to be committed and pushed?

Reply:
Hmm...there's nothing left to commit and push on my end. Is there something I can do to investigate or resolve the issue?

Reply:
perhaps rebuilding the individual target `scmake(..., force = TRUE)`?

Reply:
I may have forced a rebuild of this target when I was trying to figure out why I was getting an error when I tried to build 8_viz as a first step -- it turned out to be because the test parser I wrote didn't save lakeid as a character vector, which caused a problem at the temp merge stage. But it took me a bit to figure out where the problem was, and I likely forced a rebuild here to rule this piece out. What should I do differently next time?

Reply:
It should be fine. If this hash is different but the build file didn't change, perhaps someone else has an uncommitted build file for this? Either way, should be fine other than causing an unnecessary rebuild. @limnoliver you see other issues?

Reply:
Yep - should be fine.